build/gen_stub.php

```diff
@@ -1959,13 +1976,16 @@ public function getDeclaration(iterable $allConstInfos): string {
 
         $propertyName = $this->name->property;
 
+        $isUnknownConstValue = false;
         if ($this->defaultValue === null) {
             $defaultValue = EvaluatedValue::null();
         } else {
             $defaultValue = EvaluatedValue::createFromExpression($this->defaultValue, null, $allConstInfos);
+            $recursiveDefaultValue = EvaluatedValue::createFromExpressionRecursively($this->defaultValue, null, $allConstInfos);
```

Comment:
This is needed in order not to declare properties with such a constant default value which has an unknown constant value.